python/ray/serve/constants.py

```diff
@@ -23,7 +23,7 @@
 DEFAULT_LATENCY_SLO_MS = 1e9
 
 #: Interval for metric client to push metrics to exporters
-METRIC_PUSH_INTERVAL_S = 2
+METRIC_PUSH_INTERVAL_S = 2000
```

Comment:
Is this right? Unit says seconds.

Reply:
opps. forgot to change this back. the number was set to high from profiling. we need a way to disable metrics collection or configure this interval, under high load, 2s push interval causes metric actor CPU too high